ansible_hosts

```diff
@@ -1,5 +1,5 @@
 [local]
-127.0.0.1
+localhost ansible_connection=local
```

Comment:
There's an assumption in the old version of the playbook that this is being run on a GCE VM. I'm no longer making that assumption.

Reply:
But you did test it there as well?